Automattic/wp-calypso
# Reader: Migrate requestTags to redux. the plumbing

Part of the effort to move tags from redux to flux: https://github.com/Automattic/wp-calypso/pull/8589#issuecomment-279130631
This encompasses most of the `requestTags` work / plumbing.  It isn't hooked up to anything yet 
so it is dead code for now.

To test:
 - run `npm run test-client:watch client/state/data-layer/wpcom/read`
- sanity check that it builds + loads
